capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -226,12 +231,6 @@ function c4m_restful_quick_post_preprocess_html(&$vars) {
  *   True if current path requires the app to be loaded.
  */
 function c4m_restful_quick_post_needs_app() {
-  // Check first if the user is on the global homepage.
-  // Or if he is on a group dashboard he has no access to.
-  if (drupal_is_front_page() && !og_context()) {
```

Comment:
@HelenaEksler I just realized that maybe we can use ``og_purl_og_context_handler()`` instead of ``og_context('node', NULL, NULL, FALSE)``